jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/PointwiseDenseWithCharsModelExecutor.scala

```diff
@@ -0,0 +1,48 @@
+package ml4ir.inference.tensorflow
+
+import ml4ir.inference.tensorflow.utils.TensorUtils
+import org.tensorflow.Session
+
+class PointwiseDenseWithCharsModelExecutor(session: Session) {
+
+  val INPUT_QUERY_CHAR_NODE = "INPUT_QUERY_CHAR"
+  val INPUT_DOC_CHAR_NODE = "INPUT_DOC_CHAR"
+  val INPUT_DOC_NUMERIC_NODE = "INPUT_DOC_NUMERIC"
+  val OUTPUT_NODE = "OUTPUT"
```

Comment:
minor: shouldn't those 4 constant be in the companion object?

Reply:
yep, much better in the companion.  But this class probably needs to go away.